src/main/java/com/google/sps/data/CalendarClientData.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.*;
+
+/** Class containing the user's free hours. */
+public final class CalendarClientData {
+
+  private final int startDay;
+  private final int[] workHours;
+  private final int[] personalHours;
+
+  /**
+   * Initialize the class by calculating the start day. The free hours should be eight hours for
+   * the working hours and sixteen hours for the pesonal time initially. 
+   *
+   * @param startTime parameter of type long that gives the Unix epoch time of the start/now. The timezone is UTC
+   */
+  public CalendarClientData(long startTime) {
+    long numMillisecondsDay = 24 * 60 * 60 * 1000;
```

Comment:
Might be worth changing these variables to constants since it seems like you use them in `addEvent` as well.

Reply:
Good idea, I made this a global variable for the class.